examples/Part 02 - Intro to Private Training with Remote Execution.ipynb

```diff
@@ -36,7 +36,9 @@
   {
```

Comment:
<p>I believe the <code>if training:</code> conditional is incorrect -- if the layer isn't included at inference time, they might not do the scaling by <code>1/p</code> that's needed to correct for the dropout. TF should take care of switching to inference mode just by passing that parameter to the dropout layer as you've already done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft-TensorFlow/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I guess you are correct. I removed the conditional and checked that we get the same training/inference results from both custom and sequential models now.